### ISSUE TITLE
Missing OpenStreetMap Attribution on maps

### ISSUE THREAD

**Issue by frodrigo** [directus/directus#10671]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

An OpenStreetMap should contain an attribution as describe on https://www.openstreetmap.org/copyright

In short "© OpenStreetMap contributors" should be show on the map.

### To Reproduce

Show a map.

### Errors Shown

No error. OSM attribution is missing.

### What version of Directus are you using?

9.4.0 Docker image

### What version of Node.js are you using?

9.4.0 Docker image

### What database are you using?

9.4.0 Docker image

### What browser are you using?

Any

### What operating system are you using?

Any

### How are you deploying Directus?

Docker
